### PR TITLE
fix(examples): Improve README for http-python3.10

### PR DESCRIPTION
**Pull request by felipehuici**
Edits to Python guide, including explanation that it may be the case that apps may need more memory, and how to add more.

### CODE REVIEW

File: examples/http-python3.10/README.md [hunk 1]
```diff
@@ -17,21 +15,20 @@ kraft run -p 8080:8080 --plat qemu --arch x86_64
...
 
 If the `--plat` argument is left out, it defaults to `qemu`.
-If the `--arch` argument is left out, it default to the CPU architecture of your system.
+If the `--arch` argument is left out, it defaults to your system's CPU architecture.
 
-Once executed, it will open port `8080` and wait for connections, and it can be queried.
-
-Query the Unikraft instance using:
+Once executed, it will open port `8080` and wait for connections. To test it,
+you can use `curl`:
```
Comment (razvand) [inline examples/http-python3.10/README.md:21]: Please use one sentence per line.

File: examples/http-python3.10/README.md [hunk 2]
```diff
@@ -48,13 +45,24 @@ To close the Unikraft instance, use:
...
 
+Finally, note that depending on how you modify this example your instance *may* need more memory to run.
+To do so, use the `kraft run`'s `-M` flag, for example:
+
+```bash
+kraft run -p 8080:8080 --plat qemu --arch x86_64 -M 512M
+```
+
+## kraft and sudo
+Mixing invocations of kraft and sudo can lead to unexpected behavior.
```
Comment (razvand) [inline examples/http-python3.10/README.md:63]: Add an empty line between the heading and the text.

File: examples/http-python3.10/README.md [hunk 3]
```diff
@@ -48,13 +45,24 @@ To close the Unikraft instance, use:
...
 
-Note that closing the `kraft run` command (such as using `Ctrl+c`) does not close the Unikraft instance.
+Note that closing the `kraft run` command (e.g., via `Ctrl+c`) does not kill the Unikraft instance.
 If you want the Unikraft instance closed when closing the `kraft run` command, use the `--rm` option:
 
 ```bash
 kraft run --rm -p 8080:8080 --plat qemu --arch x86_64
 ```
 
+Finally, note that depending on how you modify this example your instance *may* need more memory to run.
```
Comment (razvand) [inline examples/http-python3.10/README.md:55]: Use `**may**` for bold instead of `*may*` for italic.

**Review by razvand — CHANGES_REQUESTED** [review]
See my comments.

File: examples/http-python3.10/README.md (hunk 1 above)
Comment (felipehuici) [reply examples/http-python3.10/README.md]: will fix.

File: examples/http-python3.10/README.md (hunk 2 above)
Comment (felipehuici) [reply examples/http-python3.10/README.md]: will fix.

File: examples/http-python3.10/README.md (hunk 3 above)
Comment (felipehuici) [reply examples/http-python3.10/README.md]: will fix.

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**